CHANGELOG.md

```diff
@@ -181,6 +181,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* [#12576](https://github.com/cosmos/cosmos-sdk/pull/12576) Remove dependency on cosmos/keyring and upgrade to 99designs/keyring v1.2.1
```

Comment:
Orthogonal to the PR proper, but this changelog entry doesn't include a `\` in front of the `#`

Reply:
Good catch! Fixed in f027a17

I think I missed this because the changelog entries in https://github.com/cosmos/cosmos-sdk/pull/12576 don't have a `\` prior to `#123` in changelog entries. The rendered Markdown looks identical so I don't think it's necessary but added here for consistency